### PR TITLE
Add new BSM type encrypted content

### PR DESCRIPTION
**Pull request by dkumanov**

### CODE REVIEW

File: version1/04 - Web services.md [hunk 1]
```diff
@@ -836,8 +836,8 @@ Content-Type: application/json
           "fieldId": string,
           "fieldName": string,
           "fieldDescription": string,
-          "fieldType": string|long|double|timestamp,
-          "fieldValues": [string|long|double|timestamp]
+          "fieldType": string|long|double|timestamp|encrypted,
+          "fieldValues": [string|long|double|timestamp|encrypted]
```
Comment (teodoradobreva) [inline version1/04 - Web services.md]: As users can't create pre-defined encrypted values, this line should be reverted.

File: version1/04 - Web services.md [hunk 2]
```diff
@@ -956,8 +956,8 @@ Content-Type: application/json
 {
   "fieldName": string,
   "fieldDescription": string,
-  "fieldType": string|long|double|timestamp,
-  "fieldValues": [string|long|double|timestamp]
+  "fieldType": string|long|double|timestamp|encrypted,
+  "fieldValues": [string|long|double|timestamp|encrypted]
```
Comment (teodoradobreva) [inline version1/04 - Web services.md]: Same, users can't create pre-defined encrypted values, so this line should be reverted.

File: version1/04 - Web services.md [hunk 3]
```diff
@@ -988,8 +988,8 @@ Content-type: application/json
   "fieldId": string,
   "fieldName": string,
   "fieldDescription": string,
-  "fieldType": string|long|double|timestamp,
-  "fieldValues": [string|long|double|timestamp]
+  "fieldType": string|long|double|timestamp|encrypted,
+  "fieldValues": [string|long|double|timestamp|encrypted]
```
Comment (teodoradobreva) [inline version1/04 - Web services.md]: Same, users can't create pre-defined encrypted values, so this line should be reverted.

**Review by teodoradobreva — CHANGES_REQUESTED** [review]

**Review by orlinDocumaster — APPROVED** [review]

File: version1/04 - Web services.md (hunk 3 above)
Comment (dkumanov) [reply version1/04 - Web services.md]: Fixed in be13583bd7c237b563a43db275fa59dfdc1f2385.

File: version1/04 - Web services.md (hunk 2 above)
Comment (dkumanov) [reply version1/04 - Web services.md]: Fixed in be13583bd7c237b563a43db275fa59dfdc1f2385.

File: version1/04 - Web services.md (hunk 1 above)
Comment (dkumanov) [reply version1/04 - Web services.md]: Fixed in be13583bd7c237b563a43db275fa59dfdc1f2385.

**Review by teodoradobreva — APPROVED** [review]

**Review by orlinDocumaster — APPROVED** [review]

**Comment by dkumanov** [comment]
Rebased on the latest develop. No other changes.

**Comment by dkumanov** [comment]
Squashed 2 commits into one in preparation for merge.

**Pull request merged**